chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -751,6 +764,42 @@ def remove_slave(slave)
       ::Kernel.system("ovs-vsctl del-port #{@nic} #{slave}")
       slave.down
     end
+
+    def up
+      slaves.each{ |s|s.up }
+      super
+    end
+
+    def destroy
+      slaves.each do |slave|
+        remove_slave(slave)
+      end
+      super
+      ::Kernel.system("ovs-vsctl del-br #{@nic}")
+      nil
+    end
+
+    def self.create(nic,slaves=[])
```

Comment:
Style/SpaceAfterComma: Space missing after comma.<br>Style/SpaceAroundEqualsInParameterDefault: Surrounding space missing in default value assignment.